app/presenters/any_link_presenter.rb

```diff
@@ -0,0 +1,92 @@
+class AnyLinkPresenter
```

Comment:
I'm not convinced that unit-tests will add much value here. The code is getting exercised in the feature specs, which is where we test the rest of the view layer.  Open to counter-arguments of course.